doc/images/HelpCard.svg

```diff
@@ -345,15 +345,15 @@
+        <text class="SVGTextShape"><tspan class="TextParagraph" font-family="Liberation Sans Narrow, sans-serif" font-size="423px" font-weight="700"><tspan class="TextPosition" x="1756" y="7861"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">Scheme </tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="8530"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">has_scheme</tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="8852"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">scheme</tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="9174"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">scheme_id</tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="9818"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">set_scheme </tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="10140"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">remove_scheme</tspan></tspan></tspan><tspan class="TextParagraph" font-family="Liberation Mono, monospace" font-size="282px" font-weight="400"><tspan class="TextPosition" x="1756" y="10462"><tspan fill="rgb(0,0,0)" stroke="none" style="white-space: pre">normalize_scheme</tspan></tspan></tspan></text>
        </g>
       </g>
       <g class="com.sun.star.drawing.CustomShape">
```

Comment:
This is something small but it would be good if these blocks with transparent backgrounds had solid white backgrounds instead. The transparent background usually renders with that chess pattern in some applications (including github) which makes it very hard to read these blocks.

Reply:
Yes I will do that in a separate commit